### ISSUE TITLE
Option to show count by file extension

### ISSUE THREAD

**Issue by GammaGames** [yycalm/linecount#5]
I think it'd be handy to have the option to see how many lines were in html, js, css, etc files seperately in the output

**Comment by alimozdemir** [comment]
I have written a solution for this feature on [#7](https://github.com/yycalm/linecount/pull/7). But, it seems owner of the project does not care about pull requests.

**Comment by GammaGames** [comment]
It looks like the last commit was on Jun 3, 2017, do you think you'd be able to post your own forked version to the extension marketplace?

**Comment by alimozdemir** [comment]
Of course, I could do that. But I don't know, is that right thing to do ? Because I just added a feature, I don't know rest of the code. If more contributer will agree on developing forked version of mine or someone else's. I/We can publish a new version of the extension on marketplace.

**Comment by GammaGames** [comment]
I'm not sure what the limitations on publishing are, but the marketplace normally has a link to the repo so I just assumed that was how people would find that it was a fork. If there are other users willing to contribute you might be able to do a community fork

**Comment by alimozdemir** [comment]
I think it is different than your thoughts. So, we will publish it with different name like "LineCount2". I think we should find more people like us to contribute the extension. Maybe, we can tag the people on pull requests to this issue and talk with them also ?

**Comment by alimozdemir** [comment]
What do you think about the future of the extension, guys @OllieDay @batjko ?

**Comment by batjko** [comment]
Quite honestly, this extension's value to me was more educational than as an actual line counter.
I think if it could be made to apply best practice everywhere, code conventions, type declarations, module organizing code base, API conventions etc... it could serve as a great Tutorial project.

**Comment by OllieDay** [comment]
> What do you think about the future of the extension, guys @OllieDay @batjko ?

Re-publishing with the PRs merged would be good!

**Comment by alimozdemir** [comment]
Thanks for replies, I have emailed the owner. If he doesn't reply in couple days, I will publish the extension with different name.

**Comment by alimozdemir** [comment]
I have published the extension. 
https://marketplace.visualstudio.com/items?itemName=lyzerk.linecounter

Please, open your PRs to my repository.
